check-vault-status.rb

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env ruby
+#
+# check-vault-status
+# 
+# DESCRIPTION:
+#   This plugins checks if vault is up & reachable. It then checks
+#   if the vault is unsealed
+#
+# OUTPUT:
+#   plain text
+#
+# PLATFORMS:
+#   Linux
+#
+# DEPENDENCIES:
+#   gem: sensu-plugin
+#   gem: vault
+#
+require 'sensu-plugin/check/cli'
+require 'vault'
+
+class VaultStatus < Sensu::Plugin::Check::CLI
+
+  option :host,
+    description: 'vault host',
```

Comment:
all these options are not lined up properly but this repo has not been setup with the standard stuff so you get a free pass on linting issues and I will fix it once I setup the skel and CI.